tests/guardrails/suspendThreshold.tests.js

```diff
@@ -1,90 +1,74 @@
+
 module.exports = (test) => {
     var screen;
-    var settings;
+    var therapyScreen;
     var screenLimit;
+    const therapySettingsValue = 75;
     it('open screen', async () => {
-        settings = await test.OpenSettingsScreen();
-        screen = await settings.OpenSuspendThresholdScreen();
+        therapyScreen = await test.OpenTherapySettingsScreen();
+        screen = await therapyScreen.OpenSuspendThresholdScreen();
         screenLimit = test.limits.suspendThreshold;
-        await screen.OpenPicker();
+        await screen.OpenPicker(therapySettingsValue);
     });
-    describe('max units at limit', () => {
-        it('can set units', async () => {
-            await screen.SwipePickerToMaxValue();
-        });
-        it('check for guardrail warning icon', async () => {
-            await expect(screen.GuardrailWarningIconPicker()).toBeVisible();
-        });
-        it('check for guardrail message', async () => {
-            await expect(screen.GuardrailMessage('High Suspend Threshold')).toBeVisible();
-        });
-    });
-    describe('max units with warning', () => {
-        it('can set units', async () => {
+    describe(name.MinimumLimit, () => {
+        it(name.SetValue, async () => {
             await screen.ApplyOne({
-                expected: { value: screenLimit.max.warning },
-                current: { value: screenLimit.max.limit },
+                expected: { value: screenLimit.min.limit },
+                current: { value: therapySettingsValue },
             });
         });
-        it('check for guardrail warning icon', async () => {
+        it(name.HasGuardrailIcon, async () => {
             await expect(screen.GuardrailWarningIconPicker()).toBeVisible();
         });
-        it('check for guardrail message', async () => {
-            await expect(screen.GuardrailMessage('High Suspend Threshold')).toBeVisible();
+        it(name.HasGuardrailMessage, async () => {
+            await expect(screen.GuardrailMessage('Low Suspend Threshold')).toBeVisible();
         });
     });
-    describe('max units with no warning', () => {
-        it('can set units', async () => {
+    describe(name.MinimumWarning, () => {
+        it(name.SetValue, async () => {
             await screen.ApplyOne({
-                expected: { value: screenLimit.max.noWarning },
-                current: { value: screenLimit.max.warning },
+                expected: { value: screenLimit.min.warning },
+                current: { value: screenLimit.min.limit },
             });
         });
-        it('check for no guardrail warning icon', async () => {
-            await expect(screen.GuardrailWarningIconPicker()).toBeNotVisible();
+        it(name.HasGuardrailIcon, async () => {
+            await expect(screen.GuardrailWarningIconPicker()).toBeVisible();
+        });
+        it(name.HasGuardrailMessage, async () => {
+            await expect(screen.GuardrailMessage('Low Suspend Threshold')).toBeVisible();
```

Comment:
[nit] Is this really the message "Low Suspend Threshold" or is this a key that maps to the actual message. If the latter, then consider wrapping like we do all other strings. So, something like `screen.LowSuspendThresholdGuardrailMessage()`?